ArduPlane/Parameters.cpp

```diff
@@ -75,7 +75,7 @@ const AP_Param::Info Plane::var_info[] = {
     // @Description: bitmask of PIDs to send MAVLink PID_TUNING messages for
     // @User: Advanced
     // @Values: 0:None,1:Roll,2:Pitch,4:Yaw
-    // @Bitmask: 0:Roll,1:Pitch,2:Yaw
+    // @Bitmask: 0:Roll,1:Pitch,2:Yaw,4:Yaw
```

Comment:
Argh! Your right I should not look at that when tired...
On Jan 6, 2016 23:02, "Tom Pittenger" <notifications@github.com> wrote:

> In ArduPlane/Parameters.cpp
> <https://github.com/diydrones/ardupilot/pull/3450#discussion_r49044330>:
>
> > @@ -75,7 +75,7 @@ const AP_Param::Info Plane::var_info[] = {
> >      // @Description: bitmask of PIDs to send MAVLink PID_TUNING messages for
> >      // @User: Advanced
> >      // @Values: 0:None,1:Roll,2:Pitch,4:Yaw
> > -    // @Bitmask: 0:Roll,1:Pitch,2:Yaw
> > +    // @Bitmask: 0:Roll,1:Pitch,2:Yaw,4:Yaw
>
> double yaw?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/diydrones/ardupilot/pull/3450/files#r49044330>.
>